src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/editdocs/EditDocsAuditService.java

```diff
@@ -24,6 +24,16 @@
         return docIds;
     }
 
+    public List<String> getUpdatedAndDeletedDocNamesForGivenField(AsylumCase asylumCase, AsylumCase asylumCaseBefore,
+                                                                AsylumCaseFieldDefinition field) {
+        List<IdValue<HasDocument>> doc = getDocField(asylumCase, field);
+        List<IdValue<HasDocument>> docBefore = getDocField(asylumCaseBefore, field);
+        docBefore.removeAll(doc);
+        List<String> docIds = new ArrayList<>();
+        docBefore.forEach(d -> docIds.add(getIdFromDocUrl(d.getValue().getDocument().getDocumentFilename())));
```

Comment:
don't think this is correct. I think we just need this here: `docIds.add(d.getValue().getDocument().getDocumentFilename()));`